docker/ci/setup.sh

```diff
@@ -127,7 +127,7 @@ EOF
                 yum -q -y install \
                     postgresql$pg-devel \
                     postgresql$pg-server \
-                    timescaledb-2-postgresql-$pg
+                    # timescaledb-2-postgresql-$pg
```

Comment:
Oof.  A catch-22.

If we can't wait for timescaledb to release pg15 support first, we can make this install this for all but 15 and also skip doctester for 15.  If we go that route, file an issue and leave a TODO comment referencing the issue.

Reply:
I updated the CI to skip running doctests/update tests (turns out both of those have a dependency on TimescaleDB). Issue: #648.